### ISSUE TITLE
Setup Prettier

### ISSUE THREAD

**Issue by webdev03** [SpellCheckJS/nodejs-client#6]
Prettier is awesome!

**Comment by webdev03** [comment]
Also, since this is a package, maybe switch to MIT?

**Comment by hello-smile6** [comment]
> Prettier is awesome!
> 
> You can make it a devDependency or just use npx since you don't like dependencies.

Actually, I'm fine with dependencies. I'm about to make https://github.com/SpellCheckJS/SpellCheckJS/ have a `package.json`, so I'll have my first dependency soon.

**Comment by webdev03** [comment]
Should I set up Prettier?

**Comment by hello-smile6** [comment]
> Should I set up Prettier?

Yes!!! Could you make it use gh actions, like your smartpage repo does?

**Comment by webdev03** [comment]
Sure. Also, what do you think about monorepo?

**Comment by hello-smile6** [comment]
> Sure. Also, what do you think about monorepo?

What's that?

**Comment by webdev03** [comment]
https://en.wikipedia.org/wiki/Monorepo you could have a global prettier config!

**Comment by webdev03** [comment]
It's similar to how Scratch mods work - clone `scratch-gui`, `scratch-vm` and everything in one repo.

**Comment by hello-smile6** [comment]
> It's similar to how Scratch mods work - clone `scratch-gui`, `scratch-vm` and everything in one repo.

Seems a bit ridiculous. Also, could you use the prettier config from Scratch Addons?

**Comment by webdev03** [comment]
Sure.

**Comment by webdev03** [comment]
```json
{
  "printWidth": 120,
  "endOfLine": "lf",
  "semi": true,
  "useTabs": false,
  "tabWidth": 2,
  "trailingComma": "none",
  "singleQuote": false
}
```
It's a modified configuration. This adds a few more specific things.

**Comment by hello-smile6** [comment]
> ```json
> {
>   "printWidth": 120,
>   "endOfLine": "lf",
>   "semi": true,
>   "useTabs": false,
>   "tabWidth": 2,
>   "trailingComma": "none",
>   "singleQuote": false
> }
> ```
> 
> It's a modified configuration. This adds a few more specific things.

Okay
